src/Microsoft.AspNetCore.Authentication.JwtBearer/LoggingExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.Logging
+{
+    internal static class LoggingExtensions
+    {
+        private static Action<ILogger, string, Exception> _tokenValidationFailed;
+        private static Action<ILogger, string, Exception> _tokenValidationSucceeded;
+        private static Action<ILogger, Exception> _errorProcessingMessage;
+
+        static LoggingExtensions()
+        {
+            _tokenValidationFailed = LoggerMessage.Define<string>(
+                eventId: 0,
+                logLevel: LogLevel.Information,
+                formatString: "Failed to validate the token {Token}.");
+            _tokenValidationSucceeded = LoggerMessage.Define<string>(
+                eventId: 0,
```

Comment:
Should make these be 0, 1, 2, etc. so that they're unique, no? (Within a given class, not across types/assemblies.)